spec/view/post_index_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'test post/index', type: :feature do
+  before(:all) do
+    @user = User.find(340)
+    @posts = Post.where(author_id: @user.id).includes(comment: [:author])
+  end
+  scenario 'posts photo test' do
+    visit user_post_index_path(340)
+    expect(page).to have_xpath("//img[@src='#{@user.photo}']")
+  end
+  scenario 'users name test' do
```

Comment:
I suggest you fix the tests so that they pass. Currently you have 29 failing, starting with this one.  Its always important to make sure that all your tests pass before submitting for review. Test ensures that your code works as expected.
See the screen shot below. It shows you all your failing tests:
<img width="1449" alt="Screenshot 2023-03-21 at 13 08 19" src="https://user-images.githubusercontent.com/48546992/226575161-76f93d5b-dabd-44f5-86ec-d72fee54b42a.png">
Before you resubmit for review, kindly run `rails spec/views` to ensure all your view specs pass
You should also strive to fix your request specs since this will serve as a terrific learning opportunity.